package.json

```diff
@@ -1,26 +1,25 @@
 {
-  "name": "nyaaya",
+  "name": "nyaaya_cms",
   "version": "0.0.0",
   "private": true,
   "dependencies": {
-    "keystone": "^0.3.19",
-    "async": "^1.5.0",
+    "async": "2.1.4",
+    "dotenv": "4.0.0",
+    "keystone": "4.0.0-beta.5",
     "lodash": "^4.13.1",
-    "moment": "^2.10.6",
-    "cloudinary": "^1.2.4",
-    "express-handlebars": "^3.0.0",
-    "handlebars": "^4.0.5",
-    "dotenv": "^2.0.0"
+    "pug": "2.0.0-beta11"
   },
   "devDependencies": {
-    "eslint": "^2.12.0",
-    "eslint-config-keystone": "^2.3.1",
+    "chai": "^3.5.0",
+    "chai-http": "^2.0.1",
+    "eslint": "3.15.0",
+    "eslint-config-keystone": "3.0.0",
     "eslint-plugin-react": "^5.1.1",
-    "express-winston": "^2.4.0",
-    "winston": "^2.4.0"
+    "mocha": "^4.0.1"
   },
   "scripts": {
     "lint": "eslint .",
-    "start": "node keystone.js"
+    "start": "node keystone.js",
+    "test" : "./node_modules/.bin/mocha tests/init.js tests/unit/**" 
```

Comment:
you don't need the `"./node_modules/.bin/mocha"` since when you run npm scripts it looks there by default. change to `"mocha tests/init.js tests/unit**"`

Reply:
nope, mocha looks by default for `test` dir, it kind of bugs me, having multiple tests in test dir hence i use this. Can change to default if this bugs you.

Reply:
oh you meant the whole path! will change that.

Reply:
resolved.